frontend/src/metabase-types/store/visualizer.ts

```diff
@@ -9,11 +9,17 @@ import type {
 export type VisualizerDataSourceType = "card";
 export type VisualizerDataSourceId = `${VisualizerDataSourceType}:${number}`;
 
+export type VisualizerDataSourceDashboard = {
+  id: number;
+  name: string;
+};
+
 export type VisualizerDataSource = {
   id: VisualizerDataSourceId;
   sourceId: number;
   type: VisualizerDataSourceType;
   name: string;
+  dashboard: VisualizerDataSourceDashboard;
```

Comment:
Let's not mix dashboards with data sources here. It also doesn't seem like we use this property anywhere anyway?